nginx.montagu.conf

```diff
@@ -168,7 +180,15 @@ server {
         proxy_redirect default;
     }
 
-    # Ideally these github bots would be on a separate proxy server
+    # metrics endpoints for prometheus
+    location /packit-api/metrics {
```

Comment:
Maybe makes more sense to flip this around as `/metrics/packit-api` and `/metrics/outpack_server`?